lib/talkative_robot.rb

```diff
@@ -0,0 +1,140 @@
+#talkative robot script with methods and classes	
+require './person'
+require './user'
+require './author'
+require './top_author'
+require './grocery_list'
+
+
+	user = User.new(name: "Lisa", age: 25, city: "Atlanta", gender: "F")
+	user.hello_user
+
+user.grandmother
+
+user.gender_greet 
+
+current_user = User.new(name: "Beth", age: 35, city: "Portland", gender: "F")
+
+author_hash = {name: "Fred", age: 25, city: "Hillsdale", gender: "M", books: " 'The Mountain and the Chasm ' ", best_book: " 'My Story' "}
+	author = Top_Author.new(author_hash)
+	author.welcome
+	author.top_seller
+
+people =[user, author]
+
+grocery_data = {
+	owner: "Bill", 
+	items: ["apples", "bananas", "cheese", "eggs", "bread", "milk"]
+}
+grocery_list = GroceryList.new(grocery_data)
+
+puts grocery_list
+
+ user = User.new({ name: "Joe", age: 42, gender: "M" })
+  #grocery_list = GroceryList.new(“grocery_list.txt”, user)
+  grocery_list.owner.name 
+
+#puts "Tell me about all the #{people}"
+
+	def age_based_message(user)
+		milestone = 75
+		if age 	+ user[:age]
+			age_delta = (milestone - age).abs
+		if age > milestone
+			puts "You turned #{milestone} #{age_delta} years ago!"
+		elsif age < milestone
+			puts "You'll turn #{milestone} in just #{age_delta} years!"		
+		else 
+			puts "Wow, you're #{milestone}!"
+		end
+	end				
```

Comment:
There shouldn't be any methods defined in this file. This file just gives commands.